constants.go

```diff
@@ -139,6 +139,9 @@ var (
 	ServerCertFile = filepath.Join(NecoDir, "server.crt")
 	ServerKeyFile  = filepath.Join(NecoDir, "server.key")
 
+	SabakanTLSCertFile = filepath.Join(SabakanDir, "sabakan-tls.crt")
+	SabakanTLSKeyFile  = filepath.Join(SabakanDir, "sabakan-tls.key")
```

Comment:
Please make it clear that they are server certificates.

```suggestion
	SabakanServerCertFile = filepath.Join(SabakanDir, "sabakan-server.crt")
	SabakanServerKeyFile  = filepath.Join(SabakanDir, "sabakan-server.key")
```